x/mongo/driver/operation.go

```diff
@@ -839,7 +821,7 @@ func (op Operation) retryable(desc description.Server) bool {
 		if op.Client != nil && (op.Client.Committing || op.Client.Aborting) {
 			return true
 		}
-		if desc.WireVersion != nil && desc.WireVersion.Max >= 6 &&
+		if desc.WireVersion != nil &&
```

Comment:
See my previous comment. I think we can remove the `desc.WireVersion != nil` piece of this conditional.